layouts/shortcodes/committers.html

```diff
@@ -0,0 +1,26 @@
+{{ $urlPre := "https://projects.apache.org" }}
+{{ $path :=  "/json/foundation" }}
+{{ $group :=  getJSON $urlPre $path "/groups.json"}}
+{{ $name := getJSON $urlPre $path "/people_name.json" }}
+{{$orgnisations := getJSON "data/teamdata.json" }}
```

Comment:
I'd prefer simpler and more descriptive variable names

```suggestion
{{ $people_names := getJSON "https://projects.apache.org/json/foundation/people_name.json" }}
{{ $team_data := getJSON "data/teamdata.json" }}
```